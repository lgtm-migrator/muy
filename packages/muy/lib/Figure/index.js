import React from "react"
import Box from "@material-ui/core/Box"

const Figure = props => (
  <Box
    component="figure"
    display="block"
    mb={3}
    ml={5}
    mr={5}
    mt={1.75}
    p={0}
    {...props}
  />
)

export default Figure
