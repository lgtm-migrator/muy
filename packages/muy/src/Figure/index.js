import React from "react"
import Box from "@material-ui/core/Box"
import makeStyles from "@material-ui/core/styles/makeStyles"

const useStyles = makeStyles(theme => ({
    root: {
        display: "block",
        marginBottom: `${theme.spacing(3)}px`,
        marginLeft: `${theme.spacing(5)}px`,
        marginRight: `${theme.spacing(5)}px`,
        marginTop: `${theme.spacing(1.75)}px`,
        padding: 0,
    },
}))

const Figure = props => {
    const classes = useStyles()
    return <Box className={classes.root} component="figure" {...props} />
}

export default Figure
