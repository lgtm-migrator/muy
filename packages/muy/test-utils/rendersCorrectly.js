import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
