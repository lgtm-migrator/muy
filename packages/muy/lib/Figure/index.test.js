import React from "react"
import rendersCorrectly from "../../test-utils/rendersCorrectly"
import Figure from "."

describe("<Figure /> field", () => {
  it("should render correctly", () => {
    rendersCorrectly(<Figure />)
  })
})
