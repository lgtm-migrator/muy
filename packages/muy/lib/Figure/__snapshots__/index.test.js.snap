// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Figure /> field should render correctly 1`] = `
<figure
  className="MuiBox-root MuiBox-root-1"
/>
`;
